resources/build.gradle.kts

```diff
@@ -13,6 +13,10 @@ plugins {
 }
 
 kotlin {
+    watchosX64()
+    watchosArm64()
+    watchosSimulatorArm64()
```

Comment:
just enable this targets in resources-build-logic/src/main/kotlin/multiplatform-library-convention.gradle.kts

Reply:
convention plugins was reworked